.github/workflows/develop-push.yml

```diff
@@ -19,54 +19,54 @@ jobs:
           --health-timeout 5s
           --health-retries 5
         ports:
-          - 5432:5432
+        - 5432:5432
+
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          ref: develop
-          token: ${{ secrets.BROADBOT_GITHUB_TOKEN }}
-      - name: Initialize Postgres DB
-        env:
-          PGPASSWORD: postgres
-        run: psql -h 127.0.0.1 -U postgres -f ./local-dev/local-postgres-init.sql
-      - name: Set up JDK 17
-        uses: actions/setup-java@v3
-        with:
-          distribution: 'temurin'
-          java-version: 17
-      # See https://github.com/actions/cache/blob/main/examples.md#java---gradle
-      - name: Cache Gradle packages
-        uses: actions/cache@v3
-        with:
-          path: |
-            ~/.gradle/caches
-            ~/.gradle/wrapper
-          key: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle') }}
-          restore-keys: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}
-      - name: Grant execute permission for gradlew
-        run: chmod +x gradlew
-      - name: Build, assemble, and test
-        id: build-assemble-test
-        run: ./gradlew build
-      - name: Upload Test Reports
-        if: always()
-        uses: actions/upload-artifact@v3
-        with:
-          name: Test Reports
-          path: build/reports
-      - name: "Bump the tag to a new version"
-        uses: databiosphere/github-actions/actions/bumper@bumper-0.1.0
-        env:
-          GITHUB_TOKEN: ${{ secrets.BROADBOT_GITHUB_TOKEN }}
-          DEFAULT_BUMP: patch
-          RELEASE_BRANCHES: develop
-          VERSION_FILE_PATH: settings.gradle
-          VERSION_LINE_MATCH: "^\\s*gradle.ext.tclVersion\\s*=\\s*'.*'"
-          VERSION_SUFFIX: SNAPSHOT
-      - name: "Publish to Artifactory"
-        run: ./gradlew artifactoryPublish
-        env:
-          ARTIFACTORY_USERNAME: ${{ secrets.ARTIFACTORY_USERNAME }}
-          ARTIFACTORY_PASSWORD: ${{ secrets.ARTIFACTORY_PASSWORD }}
-          ARTIFACTORY_REPO_KEY: "libs-snapshot-local"
+    - name: Checkout current code
+      uses: actions/checkout@v3
+      with:
+        ref: develop
+        token: ${{ secrets.BROADBOT_GITHUB_TOKEN }}
+
+    - name: Skip version bump merges
+      id: bump-skip
+      uses: ./.github/actions/bump-skip
+      with:
+        event-name: ${{ github.event_name }}
+
+    - name: Initialize Postgres DB
+      env:
+        PGPASSWORD: postgres
+      run: psql -h 127.0.0.1 -U postgres -f ./local-dev/local-postgres-init.sql
+
+    - name: Setup gradlew
+      uses: ./.github/actions/setup-gradlew
+
+    - name: Build, assemble, and test
+      id: build-assemble-test
+      run: ./gradlew build
+    - name: Upload Test Reports
+      if: always()
+      uses: actions/upload-artifact@v3
+      with:
+        name: Test Reports
+        path: build/reports
```

Comment:
we can also skip building and testing on version bumps, those will be covered by the main PR

Reply:
I am inclined to keep it 
1) Unlikely for TCL, but if we have second PR get merged w/o any conflicts and w/o a rebase - it is possible that the second PR broke functionality. Keeping this test will help us identify the breaking-change, and prevent a broken artifact from being published
2) Libraries dont have nightly tests, so these may as well function as post-merge sanity tests